util/test/sub_test

```diff
@@ -240,11 +240,30 @@ def PerfDirFile(s):
 def PerfTFile(test_filename, sfx):
     return test_filename + '.' + perflabel + sfx
 
-# read file with comments
+# Read a file or if the file is executable read its output. If the file is
+# executable, the current chplenv is copied into the env before executing.
+# Expands shell variables and strip out comments/whitespace. Returns a list of
+# string, one per line in the file.
 def ReadFileWithComments(f, ignoreLeadingSpace=True):
```

Comment:
Fairly minor, but why not throw the comments in as a docstring?

    def ReadFileWithComments(f, ignoreLeadingSpace=True):
    """
    Read a file or if the file is executable read its output. If the file is
    executable, the current chplenv is copied into the env before executing.
    Expands shell variables and strip out comments/whitespace. Returns a list of
    string, one per line in the file.
    """